README.Rmd

```diff
@@ -51,7 +51,7 @@ This will result in the REST API running at http://localhost/irods-rest/0.9.3 (o
 
 To connect to the REST API endpoint of your choice, load `rirods`, connect with `create_irods()`, and authenticate with your iRODS credentials:
 
-```{r admin, include=FALSE}
+```{r admin}
```

Comment:
Now that I think about this, did we not agree that the r package should be only user facing.

Reply:
Oh, maybe we did?    But if so, then the code cannot prep bobby... and run the docs/demo/tests...

Hmm....

Reply:
No, the code is evaluated but not shown. Or do you mean that the example is not entirely obvious from the code shown?

Reply:
if, the later we should maybe replace the `iadmin` part with `docker exec` icommands

Reply:
Okay - I think we can leave the `iadmin` functionality there, and like you said, leave it not exported in the package (requiring the  `rirods:::` prefix).